template.yaml

```diff
@@ -166,6 +171,8 @@ Conditions:
 Globals:
   Function:
     Timeout: 60
+    PermissionsBoundary: !If [CreateDefaultBoundaryPolicy, !Ref AWS::NoValue, !Ref PermissionBoundaryPolicyArn]
```

Comment:
If we use the condition name "ApplyPermissionsBoundary"  we should switch branches in this ternary expression